packages/@snowpack/plugin-workbox/src/injectManifest.ts

```diff
@@ -0,0 +1,22 @@
+import { resolve } from 'path'
+import { injectManifest as _injectManifest, InjectManifestConfig } from 'workbox-build';
+import type { SnowpackPluginFactory } from 'snowpack'
+import { pluginName, report } from './utils'
+
+export const injectManifest: SnowpackPluginFactory<InjectManifestConfig> = (_, pluginOptions = {} as InjectManifestConfig) => {
+  const { swSrc, swDest, globDirectory, ...injectManifestConfig } = pluginOptions;
+
+  if (!swSrc) throw new Error('No service worker source specified');
+  if (!swDest) throw new Error('No service worker destination specified');
```

Comment:
nit: for both of these errors, WDYT about matching the config name (`swSrc`)? That way users know they’re missing something from the options.